client/state/selectors/is-storage-addon-available.ts

```diff
@@ -0,0 +1,10 @@
+import config from '@automattic/calypso-config';
+
+/**
+ * Is the storage addon (space upgrade) available, based on config flag vs. environment.
+ *
+ * @returns {boolean} - Whether or not the storage addon is available
+ */
+const isStorageAddonAvailable = (): boolean => !! config.isEnabled( 'storage-addon' );
```

Comment:
This probably shouldn't live under `state/selectors` if it's not interacting with state directly :-)

Reply:
Cool, that makes sense. I was kind of envisioning the config as global state, but technically it isn't. Will move it into the addons folder👍

Reply:
Yeah. The selectors here don't have any connection to state, not outside of what is passed into them. So unless you are passing state into the exported function, then it probably doesn't belong here. I don't _think_ there are exceptions.